### PR TITLE
WINUI-843320 Prepared the Custom Layout UG doument content for Shimmer control in WinUI

### PR DESCRIPTION
**Pull request by GilbertmanojSF3971**
Prepared the Custom Layout UG doument content for Shimmer control in WinUI

### CODE REVIEW

File: winui/Shimmer/Custom-Layout.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+sfShimmer.CustomLayout = customLayout;
+
+Grid mainGrid = new Grid();
+mainGrid.Children.Add(sfShimmer);
+this.Content = mainGrid;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Shimmer Custom Layout in WinUI](Shimmer_images/winui_shimmer_customlayout.gif)
```
Comment (Manivannan-E) [inline winui/Shimmer/Custom-Layout.md:325]: @GilbertmanojSF3971 Custom layout in WinUI Shimmer.

File: winui/Shimmer/Custom-Layout.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+description: Learn here all about the Custom Layout support in Syncfusion WinUI Shimmer (SfShimmer) control and more.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# Custom Layout of WinUI Shimmer (SfShimmer)
+
+The `Custom Layout` property of the SfShimmer can be used to create your shimmer view. The below code demonstrates how to customize the shimmer view using the WinUI framework elements by `Custom Layout`.
+
```
Comment (Manivannan-E) [inline winui/Shimmer/Custom-Layout.md:13]: @GilbertmanojSF3971 
# Custom layout of WinUI Shimmer (SfShimmer)

The `CustomLayout` property empowers you to design your own shimmer type and giving you full control over create your desired shimmer type.

## How to create a custom shimmer type

You can create a custom shimmer layout by setting the `CustomLayout` property of the SfShimmer control. Below is a code example that demonstrates how to achieve this:

File: winui/Shimmer/Custom-Layout.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
+---
+layout: post
+title: Custom Layout in WinUI Shimmer control | Syncfusion
+description: Learn here all about the Custom Layout support in Syncfusion WinUI Shimmer (SfShimmer) control and more.
+platform: WinUI
```
Comment (Manivannan-E) [inline winui/Shimmer/Custom-Layout.md:5]: @GilbertmanojSF3971 

title: Custom layout in WinUI Shimmer control | Syncfusion
description: Learn here all about the custom layout support in Syncfusion WinUI Shimmer (SfShimmer) control to create your own shimmer type.

Comment (GilbertmanojSF3971) [reply winui/Shimmer/Custom-Layout.md:5]: @Manivannan-E Sir changed the title and description content.

File: winui/Shimmer/Custom-Layout.md (hunk 2 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Custom-Layout.md:17]: @Manivannan-E Sir , Changed the font size of the title as well as the content in the PR.

File: winui/Shimmer/Custom-Layout.md (hunk 1 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Custom-Layout.md]: @Manivannan-E Sir, Changed the image Title name.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**